sources/refund.move

```diff
@@ -117,6 +129,71 @@ module refund::refund {
         vector::destroy_empty(amounts);
     }
 
+    // === Phase 1: Timeout setup ===
+
+    public entry fun set_timeout_ts(
+        pub: &Publisher,
+        pool: &mut RefundPool,
+        timeout_ts: u64,
+        clock: &Clock,
+    ) {
+        assert_timeout_setup_phase(pool);
+        assert_publisher(pub);
+        assert!(timeout_ts > clock::timestamp_ms(clock), EInvalidTimeoutTimestamp);
+
+        option::fill(&mut pool.timeout_ts, timeout_ts);
+    }
+
+    // === Phase 2: Funding ===
+
+    /// Permissionless endpoint for funding the fund pool with the given SUI coin.
+    public entry fun fund(
+        pool: &mut RefundPool,
+        coin: Coin<SUI>,
+        ctx: &mut TxContext,
+    ) {
+        assert_funding_phase(pool);
+
+        let amount = coin::value(&coin);
+        refund_table::insert_or_add(funders_mut(&mut pool.base_pool), sender(ctx), amount);
+
+        // If overfunded then return the excess
+        let total_to_refund = total_to_refund(&pool.accounting);
+        let total_raised = total_raised_mut(&mut pool.accounting);
+
+        let is_overfunded = *total_raised + amount > total_to_refund;
+        if (is_overfunded) {
+            let excess_amount = *total_raised + amount - total_to_refund;
+            let excess = coin::split(&mut coin, excess_amount, ctx);
+
+            transfer::public_transfer(excess, sender(ctx));
+            // TODO: Improve arithmetic
+            *total_raised = total_to_refund;
+        } else {
+            *total_raised = *total_raised + amount;
+        };
+
+        balance::join(funds_mut(&mut pool.base_pool), coin::into_balance(coin)); 
+    }
+    
+    public entry fun withdraw_funds(
```

Comment:
Is this needed? Seems simpler + reasonable enough to say "once you donate funds, they cannot be withdrawn (unless there are leftovers after the claim phase)"

Reply:
This makes sense, will remove it.